pandas/tests/plotting/test_groupby.py

```diff
@@ -10,11 +10,52 @@
 import numpy as np
 
 from pandas.tests.plotting.common import TestPlotBase
+import pytest
 
 
 @td.skip_if_no_mpl
 class TestDataFrameGroupByPlots(TestPlotBase):
 
+    @pytest.mark.parametrize('equal_bins, bins, expected1, expected2', (
+        (True, 10,
+         [-3., -2.4, -1.8, -1.2, -0.6, 0., 0.6, 1.2, 1.8, 2.4],
+         [-3., -2.4, -1.8, -1.2, -0.6, 0., 0.6, 1.2, 1.8, 2.4]),
+        (True, None,
```

Comment:
Assuming the default is always 10 for `bins` then `None` is not a valid argument here and should in fact raise an error instead of giving valid results